### DISCUSSION TITLE
Why do downloaded icons have an ID?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [elrumo/macOS_Big_Sur_icons_replacements#1477] [General]
For example when I download an icon I get a filename like `5cb48f72d7a89235484af824b7af450c_Godot___Dialog.icns`. I'd rather have the filename just be icon name instead.

**Comment by kernkraft235** [comment]
You can use a mass renamer app to delete the GUID, but not every icon is named with any words. Theres also the duplicate issue. I downloaded so many icons that I really needed them to have meaningful names.

@elrumo 
If this is ever updated or fixed I suggest a schema like 

`TYPE-TARGET-SHORTDESC-FFFFFFFF` => `F-Pictures-MonoChro-A2F082C2`

where: 
	f=folder a=app d=device   
This helps cases where apps need an app icon, but also have a specific data folder
	TARGET = the XDG common name, or "useful" part of the macOS domain
	SHORTDESC = some simple chracterization of the style, like `ventura` or `monochrome`
	FFFFFFFF = 5 hex digits to represent unique user that created it, the next 3 hex is for their icon 

Example 2:
The domain name for "VMware Fusion" is `com.vmware.fusion`
 - Chop of the "com", sorting starting with the developer/company is smarter.

`a-vmware.fusion-gradient-A41CB002.icns` => app - vmware fusion -  color gradient style - InternalUserID A41CB, 2nd icon theyve made
